content/docs/deploy/core.mdx

```diff
@@ -79,57 +82,52 @@ echo "deb https://dl.cloudsmith.io/public/pomerium/pomerium/deb/debian buster ma
 
 ### Docker Image
 
-Pomerium utilizes a [minimal](https://github.com/GoogleContainerTools/distroless) [docker container](https://www.docker.com/resources/what-container). You can find Pomerium's images on [dockerhub](https://hub.docker.com/r/pomerium/pomerium). Pomerium can be pulled in several flavors and architectures.
+Pomerium also provides [Docker container images](https://www.docker.com/resources/what-container). You can find Pomerium's images on [Docker Hub](https://hub.docker.com/r/pomerium/pomerium). Pomerium can be pulled in several flavors and architectures.
 
-- `:vX.Y.Z`: which will pull the a [specific tagged release](https://github.com/pomerium/pomerium/tags).
+- `:vX.Y.Z` corresponds to a [specific tagged release](https://github.com/pomerium/pomerium/tags).
 
   ```shell-session
-  $ docker run pomerium/pomerium:v0.1.0 --version
-  v0.1.0+53bfa4e
+  $ docker run pomerium/pomerium:v0.25.0 --version
+  pomerium: 0.25.0-1704902203+e6ed4d53
+  envoy: 1.28.0+eb930e32ab5555643e09d11d490e392d0a790c5a80eb0b0ebacb1046bdbb114d
   ```
 
-- `:latest`: which will pull the [most recent tagged release](https://github.com/pomerium/pomerium/releases).
+- `:vX.Y` corresponds to the latest patch release for a specific minor version (starting with v0.25).
```

Comment:
I realize we don't have a v0.25 patch release for Core, but would it make sense to add an example? Like:
```shell-session
$ docker pull pomerium/pomerium:v25.X && docker run pomerium/pomerium:v25.X
```

Reply:
Sure, but the version output looks a little repetitive if they're all using v0.25.0. How about this, where we give the example `docker pull` command but skip the version output?